content/en/sensitive_data_scanner.md

```diff
@@ -65,14 +65,41 @@ Sensitive Data Scanner supports Perl Compatible RegEx (PCRE), but the following
 
 {{< img src="sensitive_data_scanner/sds_rules_28_03_23.png" alt="A Sensitive Data Scanner custom rule" style="width:90%;">}}
 
+### Redact sensitive data in tags
+
+If you want to redact sensitive data contained in tags, you need to [remap][3] the tag to an attribute and then redact the attribute. Make sure to disable `Preserve source attribute` in the remapper processor so that the tag isn't preserved during the remapping.
```

Comment:
```suggestion
To redact sensitive data contained in tags, you must [remap][3] the tag to an attribute and then redact the attribute. Uncheck `Preserve source attribute` in the remapper processor so that the tag is not preserved during the remapping.
```